tests/spec/fs.unlink.spec.js

```diff
@@ -104,3 +104,33 @@ describe('fs.unlink', function() {
     });
   });
 });
+
+/**
+ * fsPromises tests
+ */
+
+describe('fsPromises.unlink', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should not unlink directories', () => {
+    var fs = util.fs().promises;
```

Comment:
Maybe consider naming the variable fsPromises to make it more clear what is being tested.